honeycomb/honeycomb_test.go

```diff
@@ -177,14 +177,27 @@ func TestHoneycombOutput(t *testing.T) {
 	span.End()
 
 	assert.Equal(1, len(mockHoneycomb.Events()))
-	assert.Equal(map[string]interface{}{
-		"trace.trace_id": span.SpanContext().TraceID.String(),
-		"trace.span_id":  span.SpanContext().SpanID.String(),
-		"name":           "mySpan",
-		"attributeName":  "attributeValue",
-		"duration_ms":    1,
-		"timestamp":      mockHoneycomb.Events()[0].Timestamp, // This timestamp test isn't useful, but does let us check the whole struct
-		"service_name":   "honeycomb-test",
-	}, mockHoneycomb.Events()[0].Fields())
+	traceID := mockHoneycomb.Events()[0].Fields()["trace.trace_id"]
+	assert.Equal(span.SpanContext().TraceID.String(), traceID)
+
+	spanID := mockHoneycomb.Events()[0].Fields()["trace.span_id"]
+	assert.Equal(span.SpanContext().SpanID.String(), spanID)
+
+	name := mockHoneycomb.Events()[0].Fields()["name"]
+	assert.Equal("mySpan", name)
+
+	durationMs := mockHoneycomb.Events()[0].Fields()["duration_ms"]
+	durationMsFl, ok := durationMs.(float64)
+	assert.Equal(ok, true)
+	assert.Equal((durationMsFl > 0), true)
+
+	attributeName := mockHoneycomb.Events()[0].Fields()["attributeName"]
+	assert.Equal("attributeValue", attributeName)
+
+	timestamp := mockHoneycomb.Events()[0].Fields()["timestamp"]
+	assert.Equal(mockHoneycomb.Events()[0].Timestamp, timestamp)
```

Comment:
It sounds like from the comment [above](https://github.com/honeycombio/opencensus-exporter/pull/13/files#diff-1b891c6aa5babc3c6254e487da87ce7bL186) that we could just remove this check? Or is the comment wrong and this is actually useful to check?

Reply:
I think the comment is wrong, and that while the result of timestamp isn't particularly important, I think that it is good to check that it does exist here.